aws-arn.cabal

```diff
@@ -38,7 +38,7 @@ extra-source-files:
   README.md
 
 tested-with:
-  GHC ==8.6.5 || ==8.8.4 || ==8.10.7 || ==9.0.2 || ==9.2.1
+  GHC ==8.10.7 || ==9.0.2 || ==9.2.5 || ==9.4.5 || ==9.6.2
```

Comment:
Might be worth a wiki/handbook/asana template entry on how we do these

Reply:
OK

Reply:
https://github.com/bellroy/haskell/pull/2486